ios/StripeTerminalReactNative.swift

```diff
@@ -713,7 +713,12 @@ class StripeTerminalReactNative: RCTEventEmitter, DiscoveryDelegate, BluetoothRe
     func reader(_ reader: Reader, didFinishInstallingUpdate update: ReaderSoftwareUpdate?, error: Error?) {
         var result = Mappers.mapFromReaderSoftwareUpdate(update) ?? [:]
         if let nsError = error as NSError? {
-            result["error"] = Errors.createError(nsError: nsError)
+           let errorAsDictionary = Errors.createError(nsError: nsError)
+            // createError will return a dictionary of ["error": {the error}]
+            // so merge that with the result so we have a single result.error
+            result = result.merging(errorAsDictionary, uniquingKeysWith: { _, error in
```

Comment:
do you think we need to audit other places where we create errors? / could this happen elsewhere too?

Reply:
yep, I did a search for createError and they all were just being passed in to resolve. I didn't see any others that needed this same fix...this seemed the only case where we merge an error with other things (at least at this layer) so I think we're good 👍